### ISSUE TITLE
Snapmatic / Phone Bug (Missing Keyboard / Mouse Icons / Black UI Box)

### ISSUE THREAD

**Issue by Martovvi** [MAFINS/MenyooSP#83]
When I use snapmatic AFTER activating the trainer, that little bar with all the key bindings with the keyboard and mouse icons (grid, border, filter, zoom, self portrait, exit, take photo, etc.) it's completely black with no text and no keyboard and mouse icons. Also if you take a picture with snapmatic, you can only save it to gallery and not delete it anymore (the keyboard / mouse icons are missing there as well).

**Comment by JohnnyGuir** [comment]
> When I use snapmatic AFTER activating the trainer, that little bar with all the key bindings with the keyboard and mouse icons (grid, border, filter, zoom, self portrait, exit, take photo, etc.) it's completely black with no text and no keyboard and mouse icons. Also if you take a picture with snapmatic, you can only save it to gallery and not delete it anymore (the keyboard / mouse icons are missing there as well).

I have experienced this issue randomly without any trainer or mods.

Is it always reproducible each time you enable the trainer?

**Comment by danarreichi** [comment]
So, I have experienced this is bug from way back 2017.

**Comment by JohnnyGuir** [comment]
> So, I have experienced this is bug from way back 2017.

Did you use any trainers or mods?

**Comment by danarreichi** [comment]
> > So, I have experienced this is bug from way back 2017.
> 
> Did you use any trainers or mods?

only menyoo

**Comment by JohnnyGuir** [comment]
> > > So, I have experienced this is bug from way back 2017.
> > 
> > 
> > Did you use any trainers or mods?
> 
> only menyoo

Is it a mod related to the Snapmatic?

**Comment by danarreichi** [comment]
> > > > So, I have experienced this is bug from way back 2017.
> > > 
> > > 
> > > Did you use any trainers or mods?
> > 
> > 
> > only menyoo
> 
> Is it a mod related to the Snapmatic?

yes, it's happen everytime when I activated the mod (Pressing F8)
